Makefile

```diff
@@ -48,22 +52,57 @@ docker-build:
 		--build-arg VCS_REF=`git rev-parse --short HEAD` \
 		--build-arg BUILD_DATE=`date -u +"%Y-%m-%dT%H:%M:%SZ"` .
 
+# Get the part number from the built docker image, before the '+' symbol
+docker-release:
+	docker login
+	docker push vyper:latest
```

Comment:
These have to be changed to `docker push ethereum/vyper:lastest` And also have to be tagged locally with the exact same parameter.